docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/2234.md

```diff
@@ -0,0 +1,29 @@
+# "Custom" capabilities implementation
+
+**Prerequisites**:
+- Admin user
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Open the **PREFERENCES** tab | |
+| 4 | Click the **Launch** tab | |
+| 5 | Find the **`launch.capabilities`** preference | |
+| 6 | Input follow json as the prefrence value <ul> `{` <ul> `"testCapability1": {` <ul> `"description": "Custom test capability 1"`, <br> `"commands": [` <ul> `"echo testLine1"`, <br> `"echo 'testLine1' > ~/testFile1.txt"` </ul> `]` </ul> `},` <br> `"testCapability2": {` <ul> `"description": "Custom test capability 2",` <br> `"commands": [` <ul> `"echo testLine2",` <br> `"echo 'testLine2' > ~/testFile2.txt"` </ul> `]` </ul> `}` </ul> `}` |

```

Comment:
> `"echo 'testLine2' > ~/testFile2.txt"`

Fix to: `"echo 'testLine2' >> ~/testFile1.txt"` or change steps 20-21

Reply:
Done